### PR TITLE
Projects spacing fix

### PR DESCRIPTION
**Pull request by cdiloreto**
I committed some stuff. Thx.

### CODE REVIEW

File: src/App/Theme.js [hunk 1]
```diff
@@ -26,11 +26,21 @@ const theme = createMuiTheme({
...
   },
   typography: {
+    useNextVariants: true,
     fontFamily: '\'Karla\', sans-serif',
-    title: {
+    h2: {
       fontFamily: '\'Old Standard TT\', serif',
     },
-    display1: {
+    h3: {
```
Comment (paulmichaelxd) [inline src/App/Theme.js]: Maybe we should try to figure out how to use comma separated syntax here... Like this:
```
'h1, h2, h3, h4,h5, h6': { ... }
```

Does this not work?

File: src/Photography/Gallery.jsx [hunk 2]
```diff
@@ -57,30 +59,46 @@ Gallery.propTypes = {
...
   },
-  photoContainer: {
+  photo: {
+    width: '100%',
     height: 'auto',
-    padding: 0,
-    margin: 0,
+    verticalAlign: 'top', // Removes bottom gutter for Masonry
   },
+  // photoContainer: {
```
Comment (paulmichaelxd) [inline src/Photography/Gallery.jsx]: Please remove these comments and rebase

File: src/App/Theme.js (hunk 1 above)
Comment (cdiloreto) [reply src/App/Theme.js]: No, for some reason it's not working—can we just merge for now?

Comment (paulmichaelxd) [reply src/App/Theme.js]: Alright, its just cosmetic

**Pull request merged**